test/e2e/network/endpointslice.go

```diff
@@ -57,20 +57,36 @@ var _ = common.SIGDescribe("EndpointSlice", func() {
 		Description: The discovery.k8s.io API group MUST exist in the /apis discovery document.
 		The discovery.k8s.io/v1 API group/version MUST exist in the /apis/discovery.k8s.io discovery document.
 		The endpointslices resource MUST exist in the /apis/discovery.k8s.io/v1 discovery document.
-		API Server should create self referential Endpoints and EndpointSlices named "kubernetes" in the default namespace.
+		The cluster MUST have a service named "kubernetes" on the default namespace referencing the API servers.
+		The "kubernetes.default" service MUST have Endpoints and EndpointSlices pointing to each API server instance.
 	*/
-	framework.ConformanceIt("should have Endpoints and EndpointSlices pointing to API Server", func() {
+	framework.ConformanceIt("should have Endpoints and EndpointSlices pointing to the API Server Service", func() {
```

Comment:
I consider the conformance test in 1.22 flawed and think the check should be modified. I think changing the name of the test complicates that and isn't necessary.

Reply:
yeah, good point, renaming test is always a pain